aten/src/ATen/native/Math.h

```diff
@@ -1235,7 +1235,7 @@ C10_UNUSED inline uint8_t abs_impl(uint8_t v) {
 }
 
 template <typename T>
-inline typename std::enable_if<std::is_integral<T>::value, T>::type
+inline typename std::enable_if_v<std::is_integral_t<T>, T>
```

Comment:
```suggestion
inline typename std::enable_if_v<std::is_integral_v<T>, T>
```

Reply:
Sorry, silly typos, also `enable_if` should be `enable_if_t` :)